src/main/java/uk/gov/justice/digital/nomis/api/Caseload.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.nomis.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class Caseload {
```

Comment:
Can you explain what a caseload is? It looks like a named list of agency locations?

Reply:
A caseload a collection of agencies and determines what a user can "see" in the system. Its normally a one-to-one relationship except in the Isle of Whight and central caseload